spec/models/story_spec.rb

```diff
@@ -19,7 +19,7 @@
 
     specify do
       expect(subject.as_json['story'].keys.sort).to eq([
```

Comment:
Looks like there are not tests for the front-end changes, we should include them.